packages/shared/src/components/fields/PasswordField.tsx

```diff
@@ -55,6 +56,7 @@ export function PasswordField({
       valueChanged={(value) =>
         setPasswordStrengthLevel(passwordStrength(value).id)
       }
+      leftIcon={<LockIcon />}
```

Comment:
Is it only available in the weakest state of the password? Or could be just my understanding of this block code is wrong.

Reply:
It should be always available, I just added it to the password component to always show the lock icon on the left now.